getting-started/README.adoc

```diff
@@ -85,6 +85,8 @@ The new Kafka instance is listed in the instances table. When the instance *Stat
 [.screencapture]
 .Kafka instance options menu
 image::sak-kafka-instance-options.png[Image of Kafka instance options menu]
+
+NOTE: Only the user who creates a Kafka instance can edit or delete the instance, access the associated service account and topics, and connect to the instance. All other users can access and connect to only their own instances.
```

Comment:
@sterobin Is it worth also clarifying that you can _see_ instances created by other users in your org?

We say this in the tour:
"Although you can see instances created by other users in your organization, you can't manage or connect to those instances."

We also separately say this about service accounts:
"You can create up to two service accounts that work with your preview instance"

We could possibly have said "work only with _your_ preview instance" in that last statement

A reworked version of your note might read something like:
"Although you can see Kafka instances created by other users in your organization, you can't manage or connect to those instances. Only the user who creates an instance can edit or delete the instance, access the associated service account and topics, or connect to the instance."

Reply:
@jbyrne-redhat works for me. @jciroli-rh , wdyt? I know the second sentence still isn't second person, but I figured this note is still general enough for copy-paste purposes as needed and for consistency across multiple docs.

Reply:
I really like that, John.

Reply:
Works for me too!

Reply:
love it